recipe/build.sh

```diff
@@ -27,16 +27,16 @@ export CXX_host=$CXX_FOR_BUILD
 export AR_host=$($CC_FOR_BUILD -print-prog-name=ar)
 export LDFLAGS_host="$(echo $LDFLAGS | sed s@${PREFIX}@${BUILD_PREFIX}@g)"
 
-./configure \
+"${SRC_DIR}"/configure \
```

Comment:
just wondering if this is necesssary? Usually it isn't, and it builds without on my local machine.

Reply:
Interestingly enough, if I don't include it, every build breaks (also on Prefect) because `conda_build.sh` tries to execute `configure` from a path one level up with respect to `SRC_DIR`. If the fix needed to be somewhere else, I don't know, but this certainly fixes it.